packages/project-access/src/project/dependencies.ts

```diff
@@ -0,0 +1,12 @@
+import type { Package } from '@sap-ux/project-types';
+
+/**
+ * Helper to check for dependency/devDependency.
+ *
+ * @param packageJson - content of package.json to check
+ * @param dependency - name of the dependency
+ * @returns - true: has dependency; false: no dependency
+ */
+export const hasDependency = (packageJson: Package, dependency: string): boolean =>
+    (typeof packageJson.dependencies === 'object' && packageJson.dependencies[dependency] !== undefined) ||
```

Comment:
What are we doing here? Wouldn't `packageJson.dependencies?.[dependency]` do the very same?

Reply:
I'll change it